watertap/core/zero_order_properties.py

```diff
@@ -140,11 +129,10 @@ def define_metadata(cls, obj):
                 })
 
         obj.add_properties(
-            {'flow_vol': {'method': None},
-             'conc_mass_comp': {'method': None},
-             'cp_mass': {'method': '_cp_mass'},
-             'dens_mass': {'method': '_dens_mass'},
-             'flow_mass_comp': {'method': None}})
+            {'flow_mass_comp': {'method': None},
+             'flow_vol': {'method': '_flow_vol'},
+             'conc_mass_comp': {'method': '_conc_mass_comp'},
```

Comment:
Not for this PR, but I think it makes sense to have an additional on-demand property for mass fraction. Especially since that is essentially how the conc_mass_comp is calculated as mass_frac_comp * density.